From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 2 Mar 2015 17:24:58 +0800
Subject: [PATCH] {CONSTIFY} init/do_mounts_initrd: annotate envp_init as being
 const char*[] as it really is in init/main.c

cf dd4d9fecbeba
---
 init/do_mounts_initrd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/do_mounts_initrd.c b/init/do_mounts_initrd.c
index 1452d3c32037..bb9a5ce4d265 100644
--- a/init/do_mounts_initrd.c
+++ b/init/do_mounts_initrd.c
@@ -52,7 +52,7 @@ static void __init handle_initrd(void)
 {
 	struct subprocess_info *info;
 	static const char *const argv[] = { "linuxrc", NULL, };
-	extern char *envp_init[];
+	extern const char *envp_init[];
 	int error;
 
 	real_root_dev = new_encode_dev(ROOT_DEV);
-- 
